src/styles/main.scss

```diff
@@ -3,12 +3,28 @@
 
 #root {
     @import "containers/home";
+    background: $light-sea-green;
+}
+
+#outer-container {
```

Comment:
As it stands I don't think this will even pass the linting (regardless of being BEM or not, because we're styling by ID).

If this is a top level bunch of styles, my preference would be to create a new BEM component for it called something like: "app", which would exist in it's own file.